transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransport.kt

```diff
@@ -28,6 +29,16 @@ class MessengerTransport(
 
     constructor(configuration: Configuration) : this(configuration, DefaultTokenStore(TOKEN_STORE_KEY))
 
+    /**
+     * The product name.
+     */
+    val name = BuildKonfig.name
```

Comment:
I was thinking, maybe we should extract this values to a separate object that can be accessed statically, without a need to create an instance of MessengerTransport for that?

Reply:
Would it make sense to do this as a companion object on `MessengerTransport` instead of making a separate object?